### PR TITLE
Updated Enchants

### PR DESCRIPTION
**Pull request by DWAA1660**

### CODE REVIEW

**Comment by DWAA1660** [comment]
do u like

**Review by joshcannotcode — APPROVED** [review]
Approved

**Comment by DWAA1660** [comment]
pog ty for fixing spelling lol

**Comment by DWAA1660** [comment]
what happened to my format in the code

**Comment by srnyx** [comment]
Do we really need a list of what the enchants do? Can't the wiki just say to do `/ce` and then click on the compass?


> what happened to my format in the code

https://docs.gitbook.com/editing-content/markdown

File: mc/enchants.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # Enchants
 
+Custem enchants
+Get to menu by doing /ce
+Enchants and info
+  Sword enchants
+    Trap 1-2: Has chance to give your enemy slowness
+    Rage 1-3: Attacks will get stronger the more you keep fighting
+    Viper 1-3: has a chance to give your enemy poison
+    Snare 1-3: Has chance to give your enemy slowness and mining fatigur
```
Comment (srnyx) [inline mc/enchants.md:10]: wow josh u spelled it wrong again

**Pull request merged**